index.rst

```diff
@@ -4,9 +4,173 @@ Collimated Beam Projector Installation and ComCam Testing
+            :width: 400px
+      * - .. image:: images/CBP_point_telescope.png
+            :width: 400px
+        - .. image:: images/laser_dome.png
+            :width: 400px
+
+   **Figure 4**: The CBP and laser installed on the dome. Top row shows the CBP in the dome from different angles. Bottom row shows the CBP and telescope pointing at each other (left) and the laser installed on the dome (right).
+
+Mask Design and Installation
+----------------------------
+
+The magnification factor of the CBP mask to the ComCam image is a ratio of the focal lengths, so approximately a factor of 16.
+
+For ComCam, we left one slot empty, and we filled the remaining four slots with four different masks. One mask is a 1mm pinhole from Thorlabs, which we will use with a calibration system to calibrate the CBP to an external source, but did not use during the ComCam campaign. The other three masks are custom designed 1" photolithographic masks that we used for tests during ComCam. They are:
+
+1. A single 100 um pinhole for every amplifier on ComCam, offset so that when properly aligned the crosstalk does not intersect with any other pinhole.
+2. A single 150 um pinhole for every CCD on ComCam.
+3. A test mask with two lines of pinholes with sizes 10 um, 20 um, 50 um, 100 um, 150 um, 200 um, 250 um, 300 um, 500 um, and 1 mm.
+
+.. container:: figure
+
+   .. list-table::
+      :widths: 50 50
+      :header-rows: 0
+
+      * - .. image:: images/one_per_amp_mask.png
+            :width: 300px
+        - .. image:: images/two_line_mask.png
+            :width: 300px
+
+   **Figure 5**: The one-pinhole-per-amp mask illuminated with a blue LED (left) and the design for the mask with two lines of pinholes of varying sizes (right).
+
+
+CBP Testing with ComCam
+=======================
+
+With heroic effort from the electronics, electrical, IT, and construction teams, we were able to install the CBP and laser, hook them up, and then take data with the CBP during the last four nights of the ComCam campaign.
+
+Here is a list of the major tests and milestones we were able to achieve:
+
+1. We were able to copoint the CBP and the Simonyi Telescope.
+2. We demonstrated that the copointing locations were repeatable from night to night even as the dome, TMA, and CBP had moved during the day.
+3. We swept through focus for all of our masks.
+4. We made small movements of the CBP and TMA to see how repeatable and accurate the movements were.
+5. We performed a copointing 'dance' where we moved the CBP and telescope at the same time and kept the CBP spots on ComCam.
+6. We swept through wavelengths in the g and r bands, to find our in-band, out-of-band, and band-edge throughput. We observed a red leak in the g band.
+7. We took images to investigate nonlinearity in the crosstalk.
+8. We took images while applying different AOS bending modes and saw how the spots moved.
+9. We looked at diffraction off the channel stops from red light.
+
```

Comment:
Can we link to the list of data you have on Confluence? Or put those here for posterity?